server/templates/static/homepage.html

```diff
@@ -31,12 +31,21 @@
  
  {% block content %}
  <section id="homepage-top">
-  <div id="search-container"></div>
+  <div class="container">
+    <div class="header">Public data tells incredible stories</div>
+    <div class="sub-header">
+      Working with public data doesn't have to be painful.<br/>
+      Data Commons is a free, unified database of over
+      <a href="https://docs.datacommons.org/datasets/" target="_blank" rel="noopener noreferrer">
+        175K public datasets
```

Comment:
Not 175K public datasets :)

We can use a qualitative sentence perhaps since there are different ways to quantify the size (# variables, # sources, # datasets, # series, # data points)...

Reply:
Removed